### PR TITLE
Mobile swiper/carousel

### PR DESCRIPTION
**Pull request by DavidBachmann**
CSS only mobile swiper.

image

https://www.dropbox.com/s/xutkhsstj3wbfoa/SVIA5834.mp4?dl=0

### CODE REVIEW

File: apps/web/components/LatestNewsSection/LatestNewsSection.tsx [hunk 1]
```diff
@@ -23,8 +26,45 @@ const LatestNewsSection: React.FC<LatestNewsProps> = ({
...
+    if (width < theme.breakpoints.md) {
+      return setIsMobile(true)
+    }
+    setIsMobile(false)
+  }, [width])
+
+  const renderMobile = () => (
+    <GridContainer>
+      <GridRow>
+        <GridColumn span="12/12">
```
Comment (barabrian) [inline apps/web/components/LatestNewsSection/LatestNewsSection.tsx:42]: would it make sense to drop <GridRow> and <GridColumn> here (if the components span 100% by default)?

File: apps/web/components/LifeEventsCardsSection/LifeEventsCardsSection.tsx [hunk 2]
```diff
@@ -65,7 +80,45 @@ interface LifeEventsSectionProps {
...
-  return (
+  const { width } = useWindowSize()
+  const [isMobile, setIsMobile] = useState(false)
+
+  useIsomorphicLayoutEffect(() => {
+    if (width < theme.breakpoints.md) {
+      return setIsMobile(true)
+    }
+    setIsMobile(false)
+  }, [width])
```
Comment (barabrian) [inline apps/web/components/LifeEventsCardsSection/LifeEventsCardsSection.tsx:91]: is this something we would want to create a custom hook for?

File: apps/web/components/LatestNewsSection/LatestNewsSection.tsx (hunk 1 above)
Comment (DavidBachmann) [reply apps/web/components/LatestNewsSection/LatestNewsSection.tsx:42]: Do you mean to add that as a feature to Grid? Sounds sensible.

File: apps/web/components/LifeEventsCardsSection/LifeEventsCardsSection.tsx (hunk 2 above)
Comment (DavidBachmann) [reply apps/web/components/LifeEventsCardsSection/LifeEventsCardsSection.tsx]: Good call. I've created that.

Comment (DavidBachmann) [reply apps/web/components/LifeEventsCardsSection/LifeEventsCardsSection.tsx]: No, I change my mind. It's too messy to conditionally render something with SSR. The current strategy is now to render **both** version and hide either one with CSS.

**Review by barabrian — APPROVED** [review]
LGTM

**Pull request merged**